csm/core/services/health.py

```diff
@@ -386,6 +399,8 @@ def _get_leaf_node_health(self, health_schema, health_count_map, leaf_nodes, ale
                         self._get_leaf_node_health(value, health_count_map, leaf_nodes, alert_uuid_map, severity_val)
                     else:
                         if value:
+                            if not const.ALERT_HEALTH in value:
```

Comment:
Looks like a workaround. Can't we have a generic function which checks if all the required fields are present or not and in case, field is not present then initialize it to "NA" ?

Reply:
Done